app/controllers/billing_expenses_controller.rb

```diff
@@ -55,27 +51,8 @@ def set_volunteer
     @volunteer = Volunteer.find(params[:volunteer_id]) if params[:volunteer_id]
   end
 
-  def set_volunteer_hours
-    @volunteer_hours = @volunteer.hours.where(billing_expense: nil)
-  end
-
-  def compute_hours
-    return if @volunteer_hours.empty?
-    compute_amount(@volunteer_hours.sum(&:hours) + @volunteer_hours.sum(&:minutes)/60)
-  end
-
-  def compute_amount(hours)
-    if hours < 25
-      50
-    elsif hours < 50
-      100
-    else
-      150
-    end
-  end
-
-  def volunteer_hours_update
-    @volunteer_hours.update(billing_expense_id: @billing_expense.id)
+  def file_name
+    [@volunteer.contact.full_name, @volunteer.hours.maximum(:meeting_date)].join('-').parameterize
```

Comment:
Is the ""Spesenauszahlung-" prefix not needed anymore?

Also for clarity I would call this method `pdf_file_name`